app/src/main/java/com/sdp/movemeet/models/User.java

```diff
@@ -17,40 +16,43 @@
     private String idUser;
     private String imageId;
     private String documentPath;
+    private ArrayList<String> registeredActivities;
 
 
     /**
      * Construct a new user
-     * @param fullName : full name of the user
-     * @param email : google email of the user
-     * @param phone : phone number of the user
+     *
+     * @param fullName    : full name of the user
+     * @param email       : google email of the user
+     * @param phone       : phone number of the user
      * @param description : description of the user
      */
-    public User (String fullName, String email, String phone, String description) {
-        if(fullName == null || email == null || phone == null){
+    public User(String fullName, String email, String phone, String description, ArrayList<String> registeredActivities) {
```

Comment:
Yes! As discussed, I think it's a good idea to add this field to User objects. This will allow easy access to the list of activities the user is registered for! (Rather than downloading all the activities and then looking for each one to see if the user is registered for it! (As you might have want to implement this at the beginning)) 👍

Reply:
Yes it is easier and faster by this method ! Thanks for the advice !